Cargo.toml

```diff
@@ -19,6 +19,7 @@ reqwest = { version = "0.10", optional = true, features = ["json"] }
 futures = { version = "0.3", optional = true }
 clap = { version = "2.33", optional = true }
 base64 = { version = "^0.11", optional = true }
+bitcoincore-rpc = { git = "https://github.com/justinmoon/rust-bitcoincore-rpc", branch = "magical", optional = true }
 
 # Platform-specific dependencies
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
```

Comment:
Should I do something like this? Bitcoin RPC doesn't work in the browser ...

Reply:
I think it's fine leaving it in the normal dependencies for now, even the electrum-client doesn't work but it's still up there...